cmd/egress/cmd.go

```diff
@@ -62,8 +62,7 @@ type egressConfig struct {
 }
 
 func getDefaultRegion(platformType string) string {
-
-	if platformType == helpers.PlatformGCP {
+	if platformType == platform.GCPClassic.String() {
```

Comment:
Any reason why we shouldn't use/accept the Platform type directly instead of strings?